examples/00_basic/README.txt

```diff
@@ -0,0 +1,7 @@
+===========================
+Simple Mechanical Workflows
```

Comment:
```suggestion
Simple Mechanical workflows
```
We follow the Google developer documentation style guide, which says to use sentence case for titles. I assume "Mechanical" in this title is the Ansys product name--so am leaving this proper noun capitalized.